unic/ucd/normal/src/tables/canonical_combining_class_values.rsv

```diff
@@ -1,345 +1,345 @@
-// WARNING: Auto-generated by `tools/gen_ucd_tables.py`. DO NOT EDIT MANUALLY!
+// WARNING: Auto-generated by unic-gen. DO NOT EDIT MANUALLY!
```

Comment:
Could you please update the Python script to generate this, and use that instead, so we can maintain the gen scripts before landing `unic-gen`?